src/commands/buildCommands/configs/rules.js

```diff
@@ -0,0 +1,89 @@
+const path = require('path');
```

Comment:
Copied from [`mwp-config`](https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-config/webpack/rules.js). No edits for webpack 4.